### PR TITLE
Battery powerup

### PR DESCRIPTION
**Pull request by XerxesNoble**

### CODE REVIEW

File: src/engine.js [hunk 1]
```diff
@@ -6,29 +6,36 @@ import dispatcher from './utils/dispatcher'
...
     this.canvas = canvas
     this.context = canvas.getContext('2d')
     // Level Generation, get all objects that will be in the game
     // obstacles, monsters, powerps, player, end
     this.game = {...(map(this.canvas, this.context))}
   }
 
-  start(){
+  start() {
     this.loop();
```
Comment (Fionoble) [inline src/engine.js]: Should probably set `run = true` here.

File: src/map/levels/level-1.js [hunk 2]
```diff
@@ -7,6 +7,16 @@ export default [
...
+  '***** *  *** ******* ******* **  *** *******  ******* **',
+  '                  #                       %             ',
+  ' ***  ** ***  ** ***  ** ***  ** ***  ** ***  ** ***  **',
+  '                      %             %                   ',
+  '** ******  **** **  ****** ******  ****. * ******* *****',
+  '    #         %                                         ',
+  '******  ** **  ***   **** ***  *      ***    ***   *****',
+  '    #         %                                         ',
+  '** ***  ** ******* ******* ***  * ***     ** ***   *****',
+  ' P                **    %              #         %      ',
```
Comment (Fionoble) [inline src/map/levels/level-1.js]: Did you add all these lines because of the spacing on your resolution? We should probably set thhe level attributes to size on a pixel basis rather than a percent basis. (Can do this in another PR).

Comment (XerxesNoble) [reply src/map/levels/level-1.js]: Nah, Just so there were more levels that were actually reachable with the jump.

File: src/engine.js (hunk 1 above)
Comment (XerxesNoble) [reply src/engine.js]: ok

Comment (XerxesNoble) [reply src/engine.js]: In normal circumstances I would have done that. Since the loop is looking for a `=== false` flag.. I would say that it's unnecessary. I'll add it but there is no functionality that requires it.
